File: lua/1037.lua
n=io.read("n")
if n<0 or n>100 then print("Fora de intervalo")
elseif n<=25 then print("Intervalo [0,25]")
elseif n<=50 then print("Intervalo (25,50]")
elseif n<=75 then print("Intervalo (50,75]")
else print("Intervalo (75,100]") end

File: cpp/1037.cpp
#include<iostream>
using namespace std;

int main()
{
    float a;
    cin >> a;
    if((a < 0) or (a > 100)){
        cout << "Fora de intervalo" << endl;
    }
    else if(a > 75){
        cout << "Intervalo (75,100]" << endl;
    }
    else if(a > 50){
        cout << "Intervalo (50,75]" << endl;
    }
    else if(a > 25){
        cout << "Intervalo (25,50]" << endl;
    }
    else{
        cout << "Intervalo [0,25]" << endl;
    }
    return 0;
}